erts/emulator/beam/erl_db_util.c

```diff
@@ -2031,7 +2031,10 @@ Eterm db_prog_match(Process *c_p,
     Process *tmpp;
     Process *current_scheduled;
     ErtsSchedulerData *esdp;
-    Eterm (*bif)(Process*, ...);
+    BIF_RETTYPE (*bif_call_0)(BIF_ALIST_0);
+    BIF_RETTYPE (*bif_call_1)(BIF_ALIST_1);
+    BIF_RETTYPE (*bif_call_2)(BIF_ALIST_2);
+    BIF_RETTYPE (*bif_call_3)(BIF_ALIST_3);
```

Comment:
See erts/emulator/beam/bif.h.  It should be enough to change the type of "bif" to ErtsBifFunc (they all look the same as the parameters are passed in an array nowadays).

Reply:
Thank you! I'll get on these next round of changes.